--- p0f.fp	2016-04-16 02:51:13.000000000 +0200
+++ /etc/p0f/p0f.fp	2022-03-22 12:39:53.547770673 +0100
@@ -308,6 +308,31 @@ sig   = *:64-:0:1400:63,0:mss,ws,sok,ts,
 sig   = *:64-:0:265:31337,10:ws,nop,mss,ts,sok:ack+:0
 sig   = *:64-:0:1460:3,10:ws,nop,mss,sok,nop,nop:ecn,uptr+:0
 
+; -----
+; Scapy
+; -----
+
+label = s:!:Scapy:Default TCP() packet
+sys   = @unix,@win
+sig   = *:64:0:0:8192,0::seq-:0
+
+; -------
+; Masscan
+; -------
+
+label = s:!:Masscan:TCP scan
+sys   = @unix,@win
+sig   = *:255:0:0:1024,0:::0
+
+; ----
+; ZMap
+; ----
+
+label = s:!:ZMap:TCP scan
+sys   = @unix,@win
+sig   = *:255:0:0:65535,0:::0
+
+
 ; -----------
 ; p0f-sendsyn
 ; -----------
